LaunchPadBooster/Patching/HarmonyGameBranchPatch.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+
+namespace LaunchPadBooster.Patching
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class HarmonyGameBranchPatch : HarmonyConditionalPatch
+    {
+        public readonly String[] Branches;
+        
+        //Note: the reason for using `public` for the case where the branch is not specified is
+        //because when downloading a branch with steamcmd, when you specify `-beta public`, it will download the main branch
+        public static readonly String CurrentBranch = Steamworks.SteamApps.CurrentBetaName ?? "public";
```

Comment:
We don't really control when the static initializer happens, so there's an off chance this could run before we initialize the steam API. Could we make this a property getter instead so we can be sure it runs at the right time